jobs.py

```diff
@@ -347,14 +364,45 @@ def get_vm_or_vms_refs(handle):
         print("You need to update your local vm_data.py.")
         print("You may also bypass this error by providing your own --vm parameter(s).")
         sys.exit(1)
-    return VMS[category][key]
+
+    if type(VMS[category][key]) is list:
+        # Multi VMs
```

Comment:
"Multi VMs" could be confused with the "multi" jobs. 

Suggestion:
```suggestion
        # Version-dependent VMs
```

Reply:
Not really. `type(VMS[category][key]) is list` means `multi` category. Otherwise it's `single`. The version dependent VMS are filtered by `filter_vm()`.